inputfiles/mdn/apiDescriptions.json

```diff
@@ -183,7 +184,7 @@
   "IDBVersionChangeEvent": "The IDBVersionChangeEvent interface of the IndexedDB API indicates that the version of the database has changed, as the result of an IDBOpenDBRequest.onupgradeneeded event handler function.",
   "IIRFilterNode": "The IIRFilterNode interface of the Web Audio API is a AudioNode processor which implements a general infinite impulse response (IIR)  filter; this type of filter can be used to implement tone control devices and graphic equalizers as well. It lets the parameters of the filter response be specified, so that it can be tuned as needed.",
   "ImageData": "The ImageData interface represents the underlying pixel data of an area of a <canvas> element. It is created using the ImageData() constructor or creator methods on the CanvasRenderingContext2D object associated with a canvas: createImageData() and getImageData(). It can also be used to set a part of the canvas by using putImageData().",
-  "IntersectionObserver": "The IntersectionObserver interface of the Intersection Observer API provides a way to asynchronously observe changes in the intersection of a target element with an ancestor element or with a top-level document's viewport.",
+  "IntersectionObserver": "provides a way to asynchronously observe changes in the intersection of a target element with an ancestor element or with a top-level document's viewport.",
```

Comment:
Here the results coming from `mdn-fetch` are mangled, the very beginning of the sentence is lost.

See <a href="https://github.com/Microsoft/TSJS-lib-generator/pull/659/commits/d12ae599a8ba35075690c177094e4bb9466ac65d#r257345851">a comment above</a>